docs/advanced-topics/best-practices/wdl-best-practices.rst

```diff
@@ -1,7 +1,14 @@
 WDL Best Practices
 ==================
 
-.. include:: best-practices-intro.rst
+Best Practices
+--------------
+
+Following are some best practices for creating tools. Our intention is
+that this document will evolve as descriptor languages evolve so feel
```

Comment:
You have the word `evolve` twice in the sentence and "Our intention is..." seems unnecessary. I feel like this could read a little better, but no concrete suggestions.

Reply:
reworded